docs/releasing.md

```diff
@@ -93,8 +93,8 @@ git push --tags
 echo "Rebuild extension and viewer to get the latest, tagged master commit"
 yarn build-all;
 
-# zip the extension files, but remove lh-background as it's not needed
-cd lighthouse-extension; command rm -f dist/scripts/lighthouse-background.js; gulp package; cd ..
```

Comment:
we should really update `gulp package` to do this for us. I've definitely never remembered to delete the other files :):)

Reply:
i'll add a new PR that fixes this.

Reply:
@brendankenny see #6210